brfc_definintions.go

```diff
@@ -18,6 +18,8 @@ const (
 	BRFCSFPBuildAction                 = "189e32d93d28"       // more info: https://docs.moneybutton.com/docs/sfp/paymail-09-sfp-build.html
 	BRFCVerifyPublicKeyOwner           = "a9f510c16bde"       // more info: http://bsvalias.org/05-verify-public-key-owner.html
 	BRFCBeefTransaction                = "5c55a7fdb7bb"       // more info: https://bsv.brc.dev/payments/0070
+
+	BRFCPike = "8c4ed5ef8ace" // more info: TODO
```

Comment:
I am a big fan of creating small ticket in jira to update it, leftover TODOs are usually unfortunate and will be updated "never" :D 

I suggest to put there:

// more info: TODO -> BUX-xxx

Reply:
done